Directory.Build.props

```diff
@@ -70,7 +70,7 @@
     <IbcOptimizationDataDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'ibc'))</IbcOptimizationDataDir>
     <MibcOptimizationDataDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'mibc'))</MibcOptimizationDataDir>
     <XmlDocDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'docs'))</XmlDocDir>
```

Comment:
FYI, the TargetFramework.Sdk package also uses this property: https://github.com/dotnet/arcade/blob/2771ba1e1eb9d976b0fd9f39653be81e9883a3f1/src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/src/build/BinPlace.targets#L80

But you aren't changing it so everything's good.